draft-ietf-quic-qpack.md

```diff
@@ -164,6 +150,69 @@ unmodified.
 A string literal without a prefix length noted is an 8-bit prefix string literal
 and follows the definitions in [RFC7541] without modification.
 
+## Indexing
+
+Entries in the QPACK static and dynamic tables are addressed separately.
+
+Entries in the static table have the same indices at all times.  The static
+table is defined in Appendix A of {{!RFC7541}}. Note that because HPACK did not
+use zero-based references, there is no value at index zero of the static table.
+
+Entries are inserted into the dynamic table over time.  Each entry possesses
+both an absolute index which is fixed for the lifetime of that entry and a
+relative index which changes over time based on the context of the reference.
+The first entry inserted has an absolute index of "1"; indices
+increase sequentially with each insertion.
+
+On the control stream, a relative index of "0" always refers to the most
+recently inserted value in the dynamic table.  Note that this means the
+entry referenced by a given relative index can change while interpreting
+a HEADERS frame as new entries are inserted.
+
+~~~~~ drawing
+    +---+---------------+-------+
+    | n |      ...      | d + 1 |  Absolute Index
+    + - +---------------+   -   +
+    | 0 |      ...      | n-d-1 |  Relative Index
+    +---+---------------+-------+
+      ^                     |
+      |                     V
```

Comment:
While RFC7541 has Insertion Point on the left and Dropping Point on the right, I think it would be more intuitive if the Absolute Index grew left-to-right.

Reply:
Whatever we do, either absolute or relative will be starting at the greatest number and counting down.  Given that relative is what's sent in instructions, I found that to be more intuitive, personally.  I'm open to swapping them if most people prefer, though.

Reply:
I have a slight preference for absolute index growing left-to-right, but I can live with it as is too.